aiidalab_widgets_base/computers.py

```diff
@@ -652,72 +652,86 @@ def append_text(self, append_text):
 
 
 class ComputerDropdown(ipw.VBox):
-    """Widget to select a configured computer."""
+    """Widget to select a configured computer.
+
+    Attributes:
+        selected_computer(Unicode or Computer): Trait that points to the selected Computer instance.
+            It can be set either using Computer or computer's name. It is linked to the 'value' trait
+            of `self._dropdown` widget.
+
+        computers(Dict): Trait that contains computers found in the AiiDA database. It is linked
```

Comment:
Here I would use similar wording to the suggestion for the code dropdown

Reply:
done